config/areas/site.php

```diff
@@ -7,7 +7,7 @@
             return $kirby->site()->title()->or(t('view.site'))->toString();
```

Comment:
@bastianallgeier Do we need to update `breadcrumbLabel` too?

Reply:
No, as far as I know we only changed the menu label, which makes sense. The site title should still be visible in the breadcrumb in my opinion.